test/process_group_rpc_test.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import dist_utils
+import rpc_test
+import torch.distributed.rpc as rpc
+from dist_utils import dist_init
+
+
+def requires_process_group_agent(message=""):
+    def decorator(old_func):
+        return unittest.skipUnless(
+            dist_utils.TEST_CONFIG.rpc_backend_name == "PROCESS_GROUP", message
+        )(old_func)
+
+    return decorator
+
+
+class ProcessGroupRpcTest(rpc_test.RpcTest):
```

Comment:
This should inherit from `ProcessGroupRpcAgentTestFixture`, as it uses `rpc_agent_options` property?